### ISSUE TITLE
Will there be any news?

### ISSUE THREAD

**Issue by thytetgc** [towalink/myprox#1]
One of the most promising free projects I've seen.

It only needs a few things to be complete.

It would be great to have updates.

**Comment by towalink** [comment]
Thanks for your posting!
The project is in active use and will get updates as needed.
What "few things" would you like to see?

**Comment by thytetgc** [comment]
> Thanks for your posting! The project is in active use and will get updates as needed. What "few things" would you like to see?

It would be great if there was the possibility of connecting to Multi Clusters. Is that possible?

**Comment by towalink** [comment]
> It would be great if there was the possibility of connecting to Multi Clusters. Is that possible?

I myself could also make use of such a feature... Nevertheless, I doubt that Multi Cluster support will get implemented in MyProx in the foreseeable future. Let me explain the reasoning behind that:
Since authentication is done against Proxmox, all clusters would need to have the same user base (in practice SSO). Another issue is added complexity: Once you attempt to connect to multiple clusters, how should the system react in case one or more clusters are not reachable or respond delayed? Also: Different people will surely have different expectations of such a feature. Some might want to have something like a combobox to just be able to quickly select the cluster to work with, others will want to have a merged view of the data (VMs) of all clusters. All this would add a lot of code and therefore contradict to the desire to have a "simple" tool with a lean codebase that anybody can adapt easily, if need be.

**Comment by thytetgc** [comment]
Sure.

But today many people use multiple DCs, often in different locations and functions.

If it were possible, adding a menu to choose which Cluster, that would be great!

**Comment by towalink** [comment]
> If it were possible, adding a menu to choose which Cluster, that would be great!

Yes, adding a combox box to be able to choose the cluster to login to and to work with is indeed possible. That should not be complicated and is possibly an interesting feature for many. I'll look closer into it and will probably implement it. Thanks for your suggestion!

I have opened https://github.com/towalink/myprox/issues/2 to track this topic and will close the issue here.

**Comment by towalink** [comment]
Note: https://github.com/towalink/myprox/pull/3
